org.eclipse.tips.ui/src/org/eclipse/tips/ui/internal/TipDialog.java

```diff
@@ -43,12 +45,14 @@ public class TipDialog extends Dialog {
 	private TipComposite fTipComposite;
 	private int fShellStyle;
 	private IDialogSettings fDialogSettings;
+	private ResourceManager resourceManager;
```

Comment:
resourceManagers need to be disposed or we get non-disposed errors

Reply:
That's right, but the `resourceManager` is constructed using the two argument constructor:
`resourceManager = new LocalResourceManager(JFaceResources.getResources(), parentShell);`

So when the `parentShell` is disposed the `resourceManager` is disposed too.

Reply:
neat